operators/src/util/raster_stream_to_geotiff.rs

```diff
@@ -183,6 +188,18 @@ impl<P: Pixel + GdalType> GdalDatasetWriter<P> {
         );
         let output_bounds = query_rect.spatial_bounds;
 
+        let tile_info = tiling_specification
```

Comment:
do you only need the geotransform?  you can get that directly from the TilingStrategy. You could also add a method to the TilingSpec and reuse that also in createing the TilingStrategy.